src/main/java/com/cmlteam/serv/Serv.java

```diff
@@ -153,4 +115,72 @@ private static IllegalStateException printHelpAndExit(String message, Options op
     System.exit(1);
     return new IllegalStateException();
   }
+
+  private static String getOutputStringForMultipleFilesDownload(String urlRoot) {
+    String url = urlRoot + "dl";
+    String urlZ = url + "?z";
+    StringBuilder output = new StringBuilder();
```

Comment:
Idea tells me that this can be substituted by string:
```
StringBuilder result = new StringBuilder();
  result.append("i = ");
  result.append(i);
  result.append(";");
  return result.toString();
This code could be replaced with
  String result = "i = " + i + ";";
  return result;
```
Let's apply this in all applicable cases.

Reply:
resolved